MIDI Editor/talagan_OneSmallStep/images/note_1.lua

```diff
@@ -0,0 +1,55 @@
+-- @noindex
+-- @author Ben 'Talagan' Babut
+-- @license MIT
+-- @description This is part of One Small Step
+
+return "\z
```

Comment:
Out of curiosity, why did you encode the images as Lua files instead raw PNG blobs?

The original intent behind `CreateImageFromMem` was to allow embedding small image data directly into the script's source, or packing multiple ones in a single file. Though I guess this works too. 😄 

There's also the option of creating an image map, one big PNG and only showing portion of it each time. That should be more efficient than multiple small images too.

Reply:
It required a bit of archeology to recall why I've used this technique 😆 - I wrote OSS recently, but took some parts from an older script. I'm not sure exactly, but it's probable that I simply got inspired by the demo because it is my main source for learning (more than the API) and I copied how it was done there. Having all those little icons as lua files reminded me how we embed XPM icons in Qt/C++ projects for example, and this is a pattern I'm used too. But yeah, I admit it looks funny there ! For this one I'll probably will leave things as they are since this is working but I keep the idea of the big PNG for the next version.